src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -2747,6 +2748,13 @@ internal Conversion ClassifyConversionForCast(int position, ExpressionSyntax exp
         /// <param name="typeParameter"></param>
         public abstract ITypeParameterSymbol GetDeclaredSymbol(TypeParameterSyntax typeParameter, CancellationToken cancellationToken = default(CancellationToken));
 
+        protected BinderFlags GetSemanticModelBinderFlags()
```

Comment:
This should be internal, not protected, as it should not be part of the publicly visible API.